src/commands/Lastfm/Friends/Commands/Scrobbles.ts

```diff
@@ -19,7 +19,7 @@ export class Scrobbles extends FriendsChildCommand<typeof args> {
   idSeed = "nature sohee";
 
   description = "Shows how many scrobbles your friends have";
-  aliases = ["s"];
+  aliases = ["s", "scromples"];
```

Comment:
This is the `!friends scrobbles` command 😅

Reply:
ooops im blind fixed in 37a6997231672cd7dfdd87b18efec760947ac3f2